CGATPipelines/Pipeline/Execution.py

```diff
@@ -367,11 +367,25 @@ def run(**kwargs):
     options.update(getCallerLocals().items())
     options.update(kwargs.items())
 
-    # insert a few legacy synonyms
-    options['cluster_options'] = options.get('job_options',
-                                             options['cluster_options'])
```

Comment:
Note that this change means not that commandline cluster_options will be preferred over job_options, but that job options will be completely ignored, except for setting memory (`getJobMemory()` still parses `job_options` to see if someone has used the old `-l mem_free=([0-9]+)G` syntax for requesting memory).

Note that a quick `grep` of the repository suggests that several pipelines are still using `job_options` to set the parallel environment. These pipelines will not work after these changes (this is Okay with me, because they would already fail here anyway). We also don't know if people are using `job_options` in their own pipelines. 

cc: @AndreasHeger